types/slate/index.d.ts

```diff
@@ -1530,29 +1048,1079 @@ export interface PathUtils {
 }
 
 export interface EditorProperties {
-    onChange?: (change: Change) => void;
+    onChange?: (value: Value) => void;
     plugins?: any[];
     readOnly?: boolean;
     value?: Value;
 }
 
 export class Editor {
     object: "editor";
-    onChange: (change: Change) => void;
+    onChange: (value: Value) => void;
     plugins: any[];
     readOnly: boolean;
     value: Value;
     constructor(attributes: EditorProperties)
 
-    change(customChange: (change: Change, ...args: any[]) => Change): void;
+    /**
+     * Apply an `operation` to the editor, updating its value.
+     */
+    applyOperation(operation: Operation): Editor;
+
+    /**
+     * Synchronously flush the current changes to editor, calling onChange.
+     * In normal operation you never need to use this method! Reserved for testing.
+     */
+    flush(): Editor;
+
     command(name: string, ...args: any[]): void;
     event(handler: string, event: Event | SyntheticEvent): void;
```

Comment:
This is gone in the latest version, with the idea you would just use `run` the same way.

Reply:
Are you sure? It doesn't look to be deprecated/gone in the [latest version](https://github.com/ianstormtaylor/slate/blob/master/packages/slate/src/controllers/editor.js#L122)

Reply:
This is the line highlighted from your link:
![image](https://user-images.githubusercontent.com/289053/47809718-9642ac80-dd0f-11e8-972a-79bdc37b5a37.png)

Reply:
Sorry, I was referring to that flush method underneath https://github.com/ianstormtaylor/slate/blob/master/packages/slate/src/controllers/editor.js#L127

Reply:
Not sure how that relates to `event` not being present

Reply:
Sorry! I was looking at the wrong line for your comment, thought you'd highlighted flush in this pr